requirements_test.txt

```diff
@@ -22,6 +22,7 @@ pytest-cov<5.1.0
 pytest-memprof<0.3.0
 pytest-pyvista==0.1.8
 pytest-xdist<3.7.0
+scipy<1.14.1
```

Comment:
@tkoyama010 do you know how I can add you as an author for this? from https://github.com/pyvista/pyvista/pull/6318/commits/3596eacfc9848098ebe309f67ca782627f1a6058

Reply:
Thank you please see [Creating a commit with multiple authors](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)

Reply:
It is reflected to final squash merge if it is added to any commit.